libraries/README.md

```diff
@@ -18,12 +18,6 @@ See [github repo](https://github.com/FeralInteractive/gamemode).
 
 BSD-3-Clause licensed
 
-## hoedown
```

Comment:
shouldn't this be replaced with cmark instead of completely removed?

Reply:
Oh I forgot about that file too, changing :-)